### ISSUE TITLE
[Enhancement] flashtool: Support gathering system information (such as MCU name in the CONNECT response) without flashing.

### ISSUE THREAD

**Issue by Laikulo** [Arksine/CanBoot#85]
It would be nice to be able to get MCU type and other details from a device without actually starting a flash cycle, similar to the information provided when starting a flash.

**Comment by gpread** [comment]
I would agree this has real value, if nothing else as an easy way to determine which Katapult UUID relates to which real device/controller when you have more than one on a single canbus (which is likely since its that feature that makes CANBUS attractive in the first place).
